.github/CODEOWNERS

```diff
@@ -63,6 +63,9 @@
 # PRLabel: %Storage
 /sdk/storage/                                       @amishra-dev @rickle-msft @jaschrep-msft @gapra-msft @alzimmermsft @kasobol-msft
 
+# PRLabel: %EngSys
+/sdk/template/                                      @Azure/azure-sdk-eng
```

Comment:
Same comment on the other template. We don't support a team right now.

Reply:
@scbedd is this on the backlog for later this year?

Reply:
@joshfree Yes, and also I have a PR to implement that feature. https://github.com/Azure/azure-sdk-tools/pull/981

Reply:
Please can you go ahead and approve this. It should be fine for the automatic version increment. Can you also review https://github.com/Azure/azure-sdk-tools/pull/981 where I updated code owners to detect ID vs slug